.github/workflows/tests.yml

```diff
@@ -210,9 +210,22 @@ jobs:
       - name: Check dependencies
         run: |
           ./.ci.sh -c
-      - name: Run pytest test suite
-        run: |
-          ./.ci.sh -t
+      - name: Run pytest test suite (with code coverage)
+        run: |  # NB: We use `-v` instead of `-t` here to run this one with coverage
+          ./.ci.sh -v
+      - name: Upload API coverage to Codecov
+        uses: codecov/codecov-action@v1
```

Comment:
It looks like version 1 of `codecov-action` will stop working at the end of the month: [deprecation of v1](https://github.com/marketplace/actions/codecov#%EF%B8%8F--deprecration-of-v1). Thankfully, it looks like version 2 supports all of the flags we use ([docs](https://github.com/marketplace/actions/codecov#arguments)):
- fail_ci_if_error
- files
- flags

Reply:
Ah, good catch! I didn't even realize there was a distinction between v1 and v2. Thank you for noticing this. :D